source/mapping-rules/new-rules-suggested-mappings.txt

```diff
@@ -3,3 +3,76 @@
 ====================================
 Create Rules From Suggested Mappings
 ====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Relational Migrator can automatically suggest mapping rules based upon 
+your relational schema. Using suggested mappings can help you develop 
+your MongoDB data model faster.
+
+About this Task
+---------------
+
+In the :guilabel:`Mappings` pane, you have the option to either apply all the suggested 
+mapping rules, choose specific suggestions individually, or ignore all of the 
+suggested mapping rules. The :guilabel:`Suggested mappings` button remains visible, 
+allowing you to modify your selection at a later point.
+
+Before you Begin
+----------------
+
+- You need to enable :guilabel:`Suggested mappings` from the :guilabel:`Project Settings`. 
+  If you created a project with a recommended schema, the suggested mappings 
+  feature is disabled by default.
+
+- In order for Relational Migrator to generate suggested mappings for a specific 
+  table, you need to create a :ref:`New Documents <rm-new-documents>` 
+  mapping rule that maps the table to a collection. You can then select that 
+  collection to see the suggested mapping rules.
+
+Steps
+-----
+
+1. On the :guilabel:`Mapping` screen, click one of the following:
+
+   - A MongoDB collection on the :guilabel:`Schema model` pane.
+
+   - A collection on the MongoDB side of the project diagram.
+
+#. On the :guilabel:`Mappings` pane on the right-hand side of the screen, click 
+   the :guilabel:`Suggested mappings` button.
+   
+   The suggested mapping rules are generated from your relational schema.
+
+   .. note::
+
+      The suggested mapping rules display with a blue rectangle in the MongoDB diagram. 
+      These mapping rules are not added to your project until you click the 
+      :guilabel:`Apply` button.
+
+#. Deselect the suggested mapping rules you do not want applied to your MongoDB schema.
+
+#. Click the green :guilabel:`Apply` button at the bottom of the :guilabel:`Mappings` 
+   pane.
+
+   The suggested mapping rules are added to your project.
+
```

Comment:
(info typing)
Swap Next Steps and Learn More order, please.

Reply:
Done.